website/content/docs/concepts/api-lock.mdx

```diff
@@ -0,0 +1,82 @@
+---
+layout: docs
+page_title: Namespace API Lock
+description: Lock the Vault API on a per-namespace basis.
+---
+
+# Lock/Unlock
+
+Vault makes the API available (unlocked) by default for all namespaces. In
+this state, Vault can respond to all API/CLI ('API' from here on out) requests
```

Comment:
As a user, I might not know that Vault's CLI wrapped around its API. It might be useful to briefly mention this which would prevent the need to provide the parenthetical clarification of "'API' from here on out".

Reply:
i was mostly using this as essentially an abbreviation so that I wouldn't have to keep typing API/CLI the whole time

Reply:
That's fair, I might have been reading too much into it.